src/pybind/mgr/dashboard_v2/services/ceph_service.py

```diff
@@ -1,32 +1,112 @@
 # -*- coding: utf-8 -*-
```

Comment:
I don't like the way, you spread out `get_daemons_and_pools`. To me, your refactorization does not improve the readability, instead it makes it hard to identify that all functions and methods are only usable together and are tightly coupled to each other.